python/pypointmatcher_helper.h

```diff
@@ -0,0 +1,72 @@
+#ifndef PYTHON_PYPOINTMATCHER_HELPER_H
+#define PYTHON_PYPOINTMATCHER_HELPER_H
+
```

Comment:
The name of this file is not very clear. To me it is more like a centralized typedefs-header.

Reply:
I agree that it ended up being that way, but in the main purpose of this file was to regroup all the necessary include in one place and also the `PYBIND11_MAKE_OPAQUE` macro declaration as mentioned in pybind11 [documentation](https://pybind11.readthedocs.io/en/latest/advanced/cast/stl.html#making-opaque-types). I thought about whether I put these aliases just before their use, like the io.cpp source file, or in a common header file like it is right now. Some of them are used across some other files, so I did this to avoid re-declaring them every time I needed it. But, if you suggest that it would be better to declare it only where it is needed (saying that make me realize that it makes much more sense to do so 😅), I have no problem to fix this.

Reply:
Alright, I'll see what I can do about that.